### DISCUSSION TITLE
Why don't I have to pass the access token into the header to make requests?

### DISCUSSION THREAD

**Discussion by AlexanderCryptoOwl** [supabase/supabase#5387] [Questions] [answered]
Hi there :),

I'm new to Supabase and I'm using the API with Bash CRUD actions with a low-code tool. I'm trying to understand a few things. 

How come I don't need to pass the access_token of the user when making requests to tables? Like how does Supabase know is an authenticated user 😄 ?
![image](https://user-images.githubusercontent.com/99205909/153142948-b0f5c06a-b0a8-43de-b36c-c6759fccf7ac.png)

I enabled Row access security using the rule that only users can access my table. It doesn't seem to work, I can still easily make requests with just the header api_Key - How to make sure only auth users can SELECT * ?

![image](https://user-images.githubusercontent.com/99205909/153142589-f3205e11-c9f3-4e53-8e5b-54fa1321349b.png)

Thanks a ton :)!

**Comment by GaryAustin1** [comment, marked as answer]
If you have select policy true then that means anyone can access.  The policy is always true.  
If you use auth.role() = 'authenticated'::text then only logged in users can access (those with a jwt token as the bearer).
That all presumes your SUPABASE_KEY is the anon key and not the service_role key which skips RLS entirely.

**Reply by AlexanderCryptoOwl** [reply]
Oops thanks that solved the problem above. It works by adding the bearer + token in authentication and no longer the anon key.
